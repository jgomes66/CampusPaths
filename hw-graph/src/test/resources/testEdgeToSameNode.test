# Test Edge to Same Node
# Testing to see if node can have an edge to itself

# Create a graph
CreateGraph newGraph

# Add a node
AddNode newGraph node1

# Add an edge to node and itself
AddEdge newGraph node1 node1 e1

# List children of node1
ListChildren newGraph node1