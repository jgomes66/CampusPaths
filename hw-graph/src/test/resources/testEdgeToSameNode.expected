# Test Edge to Same Node
# Testing to see if node can have an edge to itself

# Create a graph
create graph newGraph

# Add a node
added node node1 to newGraph

# Add an edge to node and itself
added edge e1 from node1 to node1 in newGraph

# List children of node1
The children of node1 in newGraph are: node1(e1)